src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,111 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement createStatement =
+                         connection.prepareStatement(
+                             "INSERT INTO manufacturers(name, country) values(?,?);",
+                             Statement.RETURN_GENERATED_KEYS)) {
```

Comment:
You can save each query in a separate variable. Check [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-JDBC/jdbc-intro/JDBC-intro_checklist.html) and change in all methods.
```suggestion
        String insertManufacturerRequest = "INSERT INTO manufacturers(name, country) values(?, ?);";
        try (Connection connection = ConnectionUtil.getConnection();
                 PreparedStatement createStatement =
                         connection.prepareStatement(
                             insertManufacturerRequest,
                             Statement.RETURN_GENERATED_KEYS)) {
```